string NAME=takeoff

ObjectDrone d
